src/routes/cards/[cardsName]/+page.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+  import { browser } from '$app/environment'
+  import { redirect } from '@sveltejs/kit'
+  import downloadIos from '$lib/assets/apple-wallet.png'
+  import downloadAndroid from '$lib/assets/google-wallet.png'
+
+  export let data
+  const { cardData, cardNameParam } = data
+  let isSafari: boolean
+  let isIOS: boolean
+  let isAndroid: boolean
+  const oldUrl = `https://www.vermouth.nu/${cardNameParam}-vermouth-nu.pkpass`
+
+  function downloadPass() {
+    if (isIOS && isSafari) {
```

Comment:
we need to check this, what happens if I use chrome or firefox?

Reply:
I didn't verify it until you asked me to.